.circleci/config.yml

```
@@ -64,6 +67,26 @@ jobs:
           working_directory: ~/project/nrm_django
           command: |
             pipenv run black --check --diff .
+      - store_artifacts:
+          path: ./pa11y/screencaps
+      - run:
+          name: Run server for a11y tests
+          working_directory: ~/project/nrm_django
+          environment:
+            DEBUG: true
+          command: |
+            # Create admin user for Pa11y testing
+            echo "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_user('a11y_admin', 'admin@usda.gov', 'password')" | pipenv run ./manage.py shell --settings=nrm_site.settings.test
+            # Run server
+            pipenv run ./manage.py runserver 8000 --settings=nrm_site.settings.test
+          background: true
+      - run:
+          name: Run pa11y-ci accessibility check
+          working_directory: ~/project
+          command: |
+            sleep 10
+            curl --retry 10 --retry-delay 5 --retry-connrefused --silent --output /dev/null http://127.0.0.1:8000/admin
+            ! npm run test:a11y
```

This is strange enough syntax that I could use a comment on why it's there.
```suggestion
            # bang negates the return code of the accessibility tests because they don't pass.
            # TODO: when the tests pass, then this will fail. Remove the "!" when that happens.
            ! npm run test:a11y
```